runtime.txt

```diff
@@ -1 +1 @@
-python-3.8.0
+python-3.8.1
```

Comment:
@wlonk I think this should correspond with your update to OddDocker?

Reply:
Thanks, good catch. The OddDocker change was motivated by wanting to trigger getting the newer pip, so it's not gonna _behave_ differently, but this does keep things in sync.